src/Kernel32.Shared/Kernel32.cs

```diff
@@ -208,11 +208,24 @@ public enum FormatMessageFlags
         /// <param name="hObject">A valid handle to an open object.</param>
         /// <returns>
         /// If the function succeeds, the return value is nonzero.
-        /// If the function fails, the return value is zero.To get extended error information, call GetLastError.
+        /// <para>If the function fails, the return value is zero.To get extended error information, call GetLastError.</para>
         /// </returns>
         [DllImport(nameof(Kernel32), SetLastError = true)]
         public static extern bool CloseHandle(IntPtr hObject);
 
+        /// <summary>
+        /// Retrieves the thread identifier of the calling thread.
+        /// </summary>
+        /// <returns>The thread identifier of the calling thread.</returns>
+        [DllImport(nameof(Kernel32))]
+        public static extern uint GetCurrentThreadId();
```

Comment:
I suspect we should just type these as returning `int` instead of `uint`. It seems most p/invoke signatures make that decision and it makes it easier for callers to handle (such as your tests, which now have casts as a result of using `uint` here).
What do you think?